descriptors/ModuleDescriptor-template.json

```diff
@@ -1034,6 +1034,7 @@
         "circulation-storage.fixed-due-date-schedules.item.get",
         "circulation-storage.fixed-due-date-schedules.collection.get",
         "calendar.collection.get",
+        "configuration.entries.collection.get",
```

Comment:
Is this a missing permission from a previous change, or is there an extra place where we fetch configuration?

Reply:
It's a missing permission.